src/components/widgets/accounts-ratios/accounts-ratios.directive.coffee

```diff
@@ -57,6 +58,7 @@ module.controller('WidgetAccountsRatiosCtrl', ($scope, $q, $filter, $translate,
       $scope.periodInfoContext.histParams = w.metadata.hist_parameters
       $scope.periodInfoContext.accountingBehaviour = behaviour
       getPrefix(behaviour).then((label)-> $scope.periodInfoContext.injectBefore = label)
+      $scope.allowXAxisLegend = $scope.allowWidgetLegend()
```

Comment:
As this method is only being invoked here, I think we can just do this:
```
$scope.allowXAxisLegend = !_.includes(['accounts/ratios/current', 'accounts/ratios/debt_service'], w.endpoint)
```
This leverages Lodash, to shorten & add readability to the statement, and then we can remove the `allowWidgetLegend` method.